### DISCUSSION TITLE
`enginge: [knitr|jupiter]` should be globally settable

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#6539] [Feature Requests]
### Description

When dealing with the fact that inline (`R`-)code is only processed when the same `*.qmd`contains a code chunk (see #4855), I noticed that the top-level `_quarto.yml` in a project may hold global chunk options like

```yaml
execute:
  echo: false
  output: asis
```
In contrast, `engine: knitr` apparently has to (re)appear in the `yaml`-header of any project-inherent `*.qmd` file - I did, at least not manage to set that parameter globally in `_quarto.yml` (neither in the `execute`section, nor under `format: pdf:`).

It would be helpful if, analogously to above `execute:` examples, the engine could be globally defined and that definition `*.qmd`-locally overridden as needed.

**Comment by mcanouil** [comment]
See:

- https://github.com/quarto-dev/quarto-cli/issues/3157
